Cloud Pak for Data/WML/notebooks/binary/spark/Watson OpenScale and Watson ML Engine.ipynb

```diff
@@ -161,7 +162,7 @@
     "                   \"username\": \"***\",\n",
     "                   \"password\" : \"***\",\n",
     "                   \"instance_id\": \"wml_local\",\n",
-    "                   \"version\" : \"3.5\" #If your env is CP4D 4.0 then specify \"4.0\" instead of \"3.5\"\n",
+    "                   \"version\" : \"4.0\" #If your env is CP4D 4.0 then specify \"4.0\" instead of \"3.5\"\n",
```

Comment:
The comment after this needs to be changed since we are specifying 4.0 now instead of 3.5